### DISCUSSION TITLE
Branch protection for multiple repos

### DISCUSSION THREAD

**Discussion by PurityMaina** [community/community#39733] [Repositories] [answered]
### Select Topic Area

Question

### Body

Is the a way to set branch rules for multiple repos at the same time?
For instance if I have 5 repos, and I want to protect all the master branches across the 5 repos at the same time.

**Comment by leesheppard** [comment, marked as answer]
No, there are currently no branch rules available at the organization/user level to cover `settings/repository-defaults` on all child repositories.

**Comment by PurityMaina** [comment]
Thanks
